opencti-platform/opencti-front/src/private/components/workspaces/dashboards/WidgetConfig.jsx

```diff
@@ -1076,116 +1076,123 @@ const WidgetConfig = ({ workspace, widget, onComplete, closeMenu }) => {
-                                                              >
-                                                                {t_i18n(
-                                                                  capitalizeFirstLetter(
-                                                                    attribute.value,
-                                                                  ),
-                                                                )}
-                                                              </MenuItem>
-                                                            ))}
-                                                          </Select>
-                                                        );
-                                                      }
-                                                      return <div/>;
-                                                    }}
-                                                  />
-                                                </FormControl>
-                    )}
+                      && getCurrentSelectedEntityTypes(i).length > 0
+                      && (
+                      <FormControl
+                        className={classes.formControl}
+                        fullWidth={true}
+                        style={{
+                          flex: 1,
+                        }}
+                      >
+                        <InputLabel>{t_i18n('Attribute')}</InputLabel>
+                        <QueryRenderer
+                          query={stixCyberObservablesLinesAttributesQuery}
+                          variables={{
+                            elementType: getCurrentSelectedEntityTypes(i),
+                          }}
+                          render={({ props: resultProps }) => {
+                            if (resultProps
+                              && resultProps.schemaAttributeNames
+                            ) {
+                              let attributesValues = (resultProps.schemaAttributeNames.edges)
+                                .map((n) => n.node.value)
+                                .filter(
+                                  (n) => !R.includes(
+                                    n,
+                                    ignoredAttributesInDashboards,
+                                  ) && !n.startsWith('i_'),
+                                );
+                              if (
+                                attributesValues.filter((n) => n === 'hashes').length > 0
+                              ) {
+                                attributesValues = R.sort([
+                                  ...attributesValues,
+                                  'hashes.MD5',
+                                  'hashes.SHA-1',
+                                  'hashes.SHA-256',
+                                  'hashes.SHA-512',
+                                ]
+                                  .filter((n) => n !== 'hashes'));
```

Comment:
```suggestion
                                attributesValues = [
                                  ...attributesValues,
                                  'hashes.MD5',
                                  'hashes.SHA-1',
                                  'hashes.SHA-256',
                                  'hashes.SHA-512',
                                ].filter((n) => n !== 'hashes').sort();
```

It was throwing an error (try by selecting Artifact as entity type). Works well by completely remove ramda.